extensions/json-language-features/client/src/jsonClient.ts

```diff
@@ -614,3 +629,11 @@ function updateMarkdownString(h: MarkdownString): MarkdownString {
 function isSchemaResolveError(d: Diagnostic) {
 	return d.code === /* SchemaResolveError */ 0x300;
 }
+
+function getOptionsForDocument(document: TextDocument): TextEditorOptions | void {
```

Comment:
small nit: `TextEditorOptions | void` reads a bit weird. In such cases we usually use `TextEditorOptions | undefined` or `TextEditorOptions | null` and have an explicit return statement at the end of the function with the default value.